specification/communication/data-plane/MediaComposition/preview/2022-07-16-preview1/mediacomposition.json

```diff
@@ -0,0 +1,1180 @@
+          "type": "string"
+        },
+        "supportId": {
+          "description": "Id of the support input",
+          "type": "string"
+        },
+        "supportPosition": {
+          "$ref": "#/definitions/SupportPosition"
+        },
+        "supportAspectRatio": {
+          "format": "double",
+          "description": "Aspect ratio of the support stream.",
+          "type": "number"
+        }
+      }
+    },
+    "Resolution": {
+      "description": "The dimensions of the scene or objects in the scene",
+      "required": [
+        "height",
+        "width"
+      ],
+      "type": "object",
+      "properties": {
+        "width": {
+          "format": "int32",
+          "description": "Width of the object",
+          "type": "integer"
+        },
+        "height": {
+          "format": "int32",
+          "description": "Height of the object",
+          "type": "integer"
+        }
+      }
+    },
+    "RtmpMode": {
+      "description": "Rtmp mode",
+      "enum": [
+        "push",
+        "pull"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "RtmpMode",
+        "modelAsString": true
+      }
+    },
+    "RtmpStream": {
+      "description": "Rtmp stream to be used as an input or output",
+      "required": [
+        "resolution",
+        "streamKey",
+        "streamUrl"
+      ],
+      "type": "object",
+      "properties": {
+        "streamKey": {
+          "description": "Stream key of the stream",
+          "type": "string"
+        },
+        "resolution": {
+          "$ref": "#/definitions/Resolution"
+        },
+        "streamUrl": {
+          "description": "The url of the stream",
+          "type": "string"
+        },
+        "mode": {
+          "$ref": "#/definitions/RtmpMode"
+        }
+      }
+    },
+    "SrtStream": {
+      "description": "Srt stream to be used as an input or output",
+      "required": [
+        "resolution",
+        "streamUrl"
+      ],
+      "type": "object",
+      "properties": {
+        "resolution": {
+          "$ref": "#/definitions/Resolution"
+        },
+        "streamUrl": {
+          "description": "The url of the stream",
+          "type": "string"
+        }
+      }
+    },
+    "SupportPosition": {
+      "description": "Position of the support stream",
+      "enum": [
+        "topLeft",
+        "topRight",
+        "bottomLeft",
+        "bottomRight"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "SupportPosition",
+        "modelAsString": true
+      }
+    },
+    "TeamsMeeting": {
+      "description": "A Teams meeting to be used as an input or output",
+      "required": [
+        "teamsJoinUrl"
+      ],
+      "type": "object",
+      "properties": {
+        "teamsJoinUrl": {
+          "format": "uri",
+          "description": "The url from Teams to join the meeting",
+          "type": "string"
+        }
+      }
+    }
+  }
```

Comment:
You should specify a security scheme and security requirement for your operations. See the [Security section in the Azure API Style Guide](https://github.com/Azure/azure-api-style-guide/blob/main/openapi-style-guide.md#security) for more info.